### ISSUE TITLE
Error on startup. Scans all tables, then "Error: 'vpx, zip' resource not found."

### ISSUE THREAD

**Issue by markran** [mmattner/vPinBackupManagerApp#3]
I had an install that was working. Tried to manual upgrade to new version by replacing all files with new from zip. Started having error. Uninstalled, rebooted and did clean reinstall. Same error. 

```
An error occured generating game manifests, performance is degraded.
Error: 'vpx, zip' resource not found.
```

Program insta-exits on dismissing error. Log attached.
 
[vPinBackupManager_11904_20240512.log](https://github.com/mmattner/vPinBackupManagerApp/files/15287577/vPinBackupManager_11904_20240512.log)

**Comment by mmattner** [comment]
Hi I only just saw this issue, sorry for the delat.

I will take a look and roll a fix into the release I am working on at the moment. In the meantime I've provided a potential workaround in the following comment.

**Comment by mmattner** [comment]
In fact looking at the logs- the issue 'seems' to be occurring because the selected games emulator (VPX) has the file extension type set to 'vpx, zip' in pinup system settings. For now try setting it to just 'vpx'- that makes the error go away based on my testing .... but I'll add some code to cater for it. 

The image below is the screen to check in Pinup Popper Setup. Make sure 'Games File Extension' only says VPX for now

![image](https://github.com/mmattner/vPinBackupManagerApp/assets/483200/8f63bb00-c7e4-46da-aa5e-70640e610181)

To be clear, this is a bug in my code, but this workaround I think will address it until I release a fix.

**Comment by mmattner** [comment]
@markran I will close this one now as this has been fixed in version 3.0. Feel free to reach out if you have other issues.
